app/src/main/AndroidManifest.xml

```diff
@@ -16,13 +16,20 @@
         <activity android:name=".Profile3" />
         <activity android:name=".Profile2" />
         <activity android:name=".Profile1" />
-        <activity android:name=".MainActivity">
+        <activity android:name=".SplashScreen">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-
+              
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+              
+                <category android:name="android.intent.category.DEFAULT" />
```

Comment:
MainActivity used to have android.intent.category.LAUNCHER before (instead of DEFAULT now). Might be an issue. See lines 19, 23, 26 and 30.